logic/src/commonMain/kotlin/com/wire/kalium/logic/data/message/ProtoContentMapper.kt

```diff
@@ -69,15 +63,14 @@ class ProtoContentMapperImpl : ProtoContentMapper {
                     )
                 }
             }
-            is MessageContent.DeleteMessage -> {
-                GenericMessage.Content.Deleted(MessageDelete(messageId = messageContent.messageId))
-            }
-            is MessageContent.DeleteForMe -> {
-                GenericMessage.Content.Hidden(MessageHide(messageId = messageContent.messageId))
-            }
-            else -> {
-                throw IllegalArgumentException("Unexpected message content type: $messageContent")
-            }
+            is MessageContent.DeleteMessage -> GenericMessage.Content.Deleted(MessageDelete(messageId = messageContent.messageId))
+            is MessageContent.DeleteForMe -> GenericMessage.Content.Hidden(
+                MessageHide(
+                    messageId = messageContent.messageId,
+                    conversationId = messageContent.conversationId
```

Comment:
Don't we need to pass the `qualifiedConversationId` here too?


Perhaps we can do a fromTo/toFrom test for these mappers. This means: from data to protobuf and from protobuf to data, and it should match the original value.

And also do the opposite: from protobuf, to data, and then from data to protobuf, and it should match the original protobuf.


May go a bit beyond this PR, but it's something to improve in the project, it may be even very simple to automate.